lib.py

```diff
@@ -51,9 +58,10 @@ async def benchmark(ctx: commands.Context, day: int, part: int, code: bytes, ):
                 if result is not None:
                     results.append(result)
 
-            if len(results) > 0:
-                save_results(cur, op_id, day, part, code, results)
-                db.commit()
+            if results:
+                with Database() as db:
+                    # TODO dont hardcode 2023
+                    db.save_results(op_id, 2023, day, part, code, results)
```

Comment:
There's already a `year()` function in this file, but OK.

Reply:
Same as above